script/deploy.js

```diff
@@ -471,19 +475,19 @@ async function main() {
       const options = program.opts();
       let config = await loadConfiguration(options.config);
       delete config.staging; // allows get asked for this one
-      await deployAirdropReservations(config);
+      await deployAirdrop(config);
       await saveConfiguration(config);
     });
 
-    program
+  program
     .name('deploy-airdrop-reservations')
     .command('deploy-airdrop-reservations')
     .description('Interactively deploys the RecurringGrantDropReservations contracts.')
     .action(async () => {
       const options = program.opts();
       let config = await loadConfiguration(options.config);
       delete config.staging; // allows get asked for this one
-      await deployAirdrop(config);
+      await deployAirdropReservations(config);
```

Comment:
the deploy script above and this one had their functions incorrectly swapped. i fixed it here